pipelines/rh-push-to-registry-redhat-io/README.md

```diff
@@ -0,0 +1,28 @@
+# rh-push-to-registry-redhat-io pipeline
+
+Tekton pipeline to push images to a registry.redhat.io registry.
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| release | The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution | No | - |
+| releaseplan | The namespaced name (namespace/name) of the releasePlan | No | - |
+| releaseplanadmission | The namespaced name (namespace/name) of the releasePlanAdmission | No | - |
+| releasestrategy | The namespaced name (namespace/name) of the releaseStrategy | No | - |
```

Comment:
According to [PR#273](https://github.com/redhat-appstudio/release-service/pull/273) once it's merged then RleaseStrategy won't be supported anymore, isn't better we prepare this pipeline to meet that PR ?

Reply:
Yes, +1